src/sdl2_ui.cpp

```diff
@@ -334,6 +329,12 @@ bool Sdl2Ui::RefreshDisplayMode() {
 			Output::Debug("SDL_GetRendererInfo failed : %s", SDL_GetError());
 		}
 
+#ifdef EMSCRIPTEN
+		// Emscripten owns the main loop.
+		SetFrameRateExternallyManaged(true);
```

Comment:
Due to ASYNCIFY magic this is not needed anymore and can use the standard logic.